rholang/src/test/scala/coop/rchain/rholang/interpreter/accounting/CostAccountingSpec.scala

```diff
@@ -277,6 +264,22 @@ class CostAccountingSpec extends FlatSpec with Matchers with PropertyChecks with
     }
   }
 
+  "replay the same term with not enough phlo" should "get the same result" in {
+    val r = scala.util.Random
+    for (_ <- 1 to 100) {
```

Comment:
Here are many tests removed because only 100 terms are considered. Each number represents a unique term so reducing the number of iterations means skipping tests for all these Rholang terms.

Reply:
If you look at the generated terms, they are almost identical with minus changes.

Reply:
Not neccessary to do all for 10000 times to make the test run much longer.

Reply:
Yes, that's the point of this test, to check all possible combinations with very small deviations. So I'd say that running first 100 is not necessary at all because these basic terms are already covered by Rholang existing tests.

Maybe we can add another generator with terms without same channels in joins. But in a way to not just collapse joins on the same channels because that means we will generate and run the same terms for different input numbers.